package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@git-town/action",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">=20.0.0"
+  },
+  "engineStrict": true,
+  "scripts": {
+    "dev": "npm run build --watch",
+    "build": "esbuild src/index.ts --outfile=dist/index.js --bundle --minify --platform=node --target=node20",
+    "test": "vitest",
+    "prepare": "husky"
+  },
+  "devDependencies": {
+    "@actions/core": "^1.10.1",
+    "@actions/exec": "^1.1.1",
+    "@actions/github": "^6.0.0",
+    "@types/node": "^20.11.24",
+    "@vercel/style-guide": "^5.2.0",
+    "editorconfig-checker": "^5.1.5",
+    "esbuild": "^0.20.0",
+    "eslint": "^8.57.0",
+    "eslint-plugin-prettier": "^5.1.3",
+    "graphology": "^0.25.4",
+    "graphology-traversal": "^0.3.1",
+    "husky": "^9.0.11",
+    "lint-staged": "^15.2.2",
+    "prettier": "^3.2.5",
+    "remark": "^15.0.1",
+    "remark-gfm": "^4.0.0",
+    "toml": "^3.0.0",
```

Comment:
Should this be a `dependency`? It is listed as a `devDependency` right now. Separating the two types of dependencies would allow us to use `npm ci --production` on CI. That's significantly snappier than `npm ci`.

Reply:
It would be only a semantic difference, as the dependencies are bundled by `esbuild` prior to the action running, removing the need for `npm ci --production` when running the action.

I do agree though that separating out the dependencies makes it clearer what is a runtime dependency, and what is tooling/dev related. Unfortunately there's no way to install only dev dependencies, so we can't optimise installs for checks that don't need dependencies to be installed.

Reply:
Interesting, thanks for the background. I have to get used to this esbuild bundle thing.